packages/evm/src/fungible.ts

```diff
@@ -0,0 +1,104 @@
+import type {
+  Domain,
+  EvmResource,
+  FeeHandlerType,
+} from "@buildwithsygma/sygma-sdk-core";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+interface TransactionRequest {
+  to: string;
+  value: bigint;
+  data: string;
+  gasLimit: bigint;
+}
+
+interface Eip1193Provider {
+  request(request: {
+    method: string;
+    params?: Array<unknown> | Record<string, unknown>;
+  }): Promise<unknown>;
+}
+
+type EvmFee = {
+  fee: bigint;
+  type: FeeHandlerType;
+  handlerAddress: string;
+  tokenAddress?: string;
+  percentage?: number;
+  minFee?: bigint;
+  maxFee?: bigint;
+};
+
+/**
+ * Return amount of liqudiity tokens on resource handler
+ * @param provider
+ * @param resource
+ */
+export function getLiquidity(
+  provider: Eip1193Provider,
+  resource: EvmResource,
+): Promise<bigint> {
+  throw new Error("Method not implemented");
+}
+
+export function createEvmFungibleAssetTransfer(
```

Comment:
is spectre part of the release, if so how would that be specified

Reply:
Added SecurityModel params. But only option is MPC. Once we add support for spectre, it will only change SDK code paths